rollup.config.js

```diff
@@ -45,7 +45,21 @@ const plugins = (minify) =>
 
 export default [
   // UMD build
-  // dist/chart.umd.js
+  // dist/chart.umd.min.js
+  {
+    input: 'src/index.umd.ts',
+    plugins: plugins(true),
+    output: {
+      name: 'Chart',
+      file: 'dist/chart.umd.min.js',
+      format: 'umd',
+      indent: false,
+      sourcemap: true,
+    },
+  },
+
+  // UMD build
+  // dist/chart.umd.js (old filename, deprecated)
```

Comment:
I wouldn't say it's deprecated. Then we will have a file rename again in V5 and people weren't happy about this one either so I would just let them co exist and not mark any of them deprecated.

@etimberg @kurkle what are your thoughts about this?

Reply:
Yeah, I wouldn't deprecate this now.

Reply:
Undeprecated. Not a big deal, it can always be deprecated later.